terraform/addons/mdm/README.md

```diff
@@ -58,7 +58,7 @@ No requirements.
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.53.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.53.1 |
```

Comment:
is this a hit we missed last time?  this isn't a fleet version

Reply:
```suggestion
| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.53.0 |
```

Reply:
reverted